modules/issue_tracker/jsx/WatchingCheckbox.js

```diff
@@ -0,0 +1,36 @@
+class WatchingCheckbox extends React.Component {
+  constructor(prop) {
+    super();
+    this.state = {
+      checked: prop.checked,
+      issueId: prop.issue_id
+    };
+    this.onChange = this.onChange.bind(this);
+  }
+  render() {
+    return (
+      <td>
```

Comment:
Remove this `<td>`

![image](https://cloud.githubusercontent.com/assets/6627543/24982706/d7a3ce2a-1fb0-11e7-94c3-2688a2ba0d69.png)